### PR TITLE
Minor examples fixes

### PR DESCRIPTION
**Pull request by floatn**

### CODE REVIEW

File: examples/basics/multi_output_example.exs [hunk 1]
```diff
@@ -18,13 +18,13 @@ data = Stream.repeatedly(fn ->
...
+  |> Axon.dense(32, activation: :relu)
+  |> Axon.dense(64, activation: :relu)
 
-fc = Axon.dense(input, 32, activation: :relu)
-fc = Axon.dense(fc, 64, activation: :relu)
-
-out1 = Axon.dense(fc, 1)
-out2 = Axon.dense(fc, 1)
+out1 = Axon.dense(model, 1)
+out2 = Axon.dense(model, 1)
```
Comment (seanmor5) [inline examples/basics/multi_output_example.exs:27]: The way this model was written was intentional to show that a multi-output model is actually just separate Axon objects in a container.

Comment (floatn) [reply examples/basics/multi_output_example.exs:27]: Now I see what `c` in `fc` did stand for. :)
Isn't `model = {out1, out2}` line what represents the container?

Comment (seanmor5) [reply examples/basics/multi_output_example.exs:27]: Yes, correct!

Comment (floatn) [reply examples/basics/multi_output_example.exs:27]: Are you suggesting that current PR changes would conceal this aspect?

Comment (seanmor5) [reply examples/basics/multi_output_example.exs:27]: Yes, I don't think it's necessary

**Comment by floatn** [comment]
@seanmor5
Can you help me with fixing onnx_check failure? I don't understand what it's error message is about.

**Comment by seanmor5** [comment]
It is upstream!

**Pull request merged**